packages/react/src/themes/types.ts

```diff
@@ -209,7 +209,18 @@ export interface ICSSPseudoElementStyle extends ICSSInJSStyle {
   content?: string
 }
 
-export interface ICSSInJSStyle extends React.CSSProperties {
+export interface StardustAnimationName {
+  keyframe?: any
+  params?: object
+}
+
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
+
+export type CSSProperties = Omit<React.CSSProperties, 'animationName'> & {
+  animationName?: StardustAnimationName | string | 'none'
+}
+
+export interface ICSSInJSStyle extends CSSProperties {
```

Comment:
This makes the ICSSInJSStyle not being able to accept something typed as React.CSSProperties. Not sure if this will be a problem, but if we don't add this, we will have to introduce new names for the animations defined as stardust animations (as `{keyframe: function, params: {...}}`)